### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by KrishnaSSH** [FossifyOrg/General-Discussion#138] [Ideas]
Hey,
I am a web developer, I'd be happy to contribute to your project and its continued development. I've reviewed the project and believe my skills in web development could be valuable. I see that the website is currently is in development i can contribute to the process of web development as I'm interested in exploring contribution opportunities.
Please let me know how I can best get involved based on the project's current needs.

**Comment by naveensingh** [comment]
Hi!

I was going to reply to your email but oh well. The website is developed by @ronniedroid and was pretty much finished last time I checked. It's not live yet because I'm yet to review and merge the [PR](https://github.com/FossifyOrg/fossifyorg.github.io/pull/6). If you want, you can help with the apps' metadata for the website, more info regarding that is [here](https://github.com/FossifyOrg/fossifyorg.github.io/issues/1#issuecomment-1890385714). If you have any suggestions regarding the website's UI/UX, feel free to post them here or in [this thread](https://github.com/FossifyOrg/fossifyorg.github.io/issues/1).

Other than that, if you have any Android development experience, you can try working on some existing issues. I'll notify you if and when we need any help with the website.

--- 

Hey @ronniedroid, do we need any help with the website?

**Reply by KrishnaSSH** [reply]
As per your inquiry regarding my experience in Android development, I regret to inform that I do not possess such expertise.

**Comment by ronniedroid** [comment]
@naveensingh not really no, only the blog is left and that is really easy to implement, I am just busy right now.
The website repo needs some contribution (mostly translation) documentation too. 

As you said, @Krishna-G-OP can help with the app meta-data and documentation if he likes.

**Reply by KrishnaSSH** [reply]
Sure, I'm down to help with the website documentation!

**Comment by naveensingh** [comment]
@ronniedroid Ok, thanks. Also, fix that spelling now!
